clvm/operators.py

```diff
@@ -53,15 +62,13 @@ def f(args):
                 return self.unknown_op_handler(op, args)
         return f(arguments)
 
-    def set_unknown_op_handler(self, callback: Callable[[bytes, CLVMObject], Tuple[int, CLVMObject]]):
```

Comment:
I suppose this is a poor-man's way of making it immutable, probably good enough

Reply:
Yes, that's the intent. It's not really, of course, but these are only instantiated in a few places.